### ISSUE TITLE
Lab 19 very confusing and hard to follow

### ISSUE THREAD

**Issue by joannex** [MicrosoftLearning/SC-300-Identity-and-Access-Administrator#107]
# Module: 4
### Task: 02
Intro unclear
Step 1 I don't think is a step
Step 2 Unnecessary as you are already in the ap.
Step 5 Unclear what to do if anything - no detail as to what app to select.

### Task: 05 (Should be 04?)
Steps 1 and 2 unnecessary if following through the lab as you are already there. 
Step 5 the suggested alternative URI doesn't work.  Possibly better to simply accept the default.

Exercise 2 - is this necessary - I consider this to be more relevant to enterprise app mgmt (but could be wrong). I think previously the lab got you to create an app role within the app which I consider to be nicer to do....then this could be followed through by going to the enterprise app, adding a user and seeing the role you have just created.

**Comment by awillslo** [comment]
http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https%3A%2F%2Fvault.azure.net
url does not work. Thanks for researching @R-C-Stewart.
